modules/ROOT/pages/apikit-odatav4-expand-feature.adoc

```diff
@@ -0,0 +1,98 @@
+= Implement the $expand Feature
```

Comment:
This title sound a bit weird. What about "Implement $expand using _generic expand_"?

Reply:
The title "Implement $expand using generic expand" assumes that you know the meaning of "$expand" and the meaning of "generic expand". In a perfect world, we would have:

- APIkit
  - APIkit for OData
    - Create an OData v4 API with APIkit
      - The $expand feature
      - The general expand feature
      - Implement $expand using generic expand

Actually, the editor suggested that we should split this article into two or three when we rearrange the APIkit nav tree. So this may be a great idea for the near future. 
What do you think?

Reply:
All in for splitting the article at some future point 🚀

Reply:
Absolutely!
I'll be contacting you so you can help me with such tasks! hahaha
Thanks!